src/ServiceControlInstaller.PowerShell/ServiceControlInstaller.Powershell.dll-help.xml

```diff
@@ -630,7 +678,7 @@ Get-SecurityIdentifier MyDomain\Foo</dev:code>
 			<command:parameter required="true" variableLength="false" globbing="false" pipelineInput="true (ByValue, ByPropertyName)" position="0">
 				<maml:name>File</maml:name>
 				<maml:description>
-					<maml:para>The path to the license file</maml:para>
+					<maml:para>The path to the license file that will be imported. The license file is written to the follwoing locations in the regisrty:  HKEY_LOCAL_MACHINE\SOFTWARE\ParticularSoftware\License</maml:para>
```

Comment:
Spelling of `Registry`

`locations` is plural but only one location is specified